performance/tests/Item20Items.test.js

```diff
@@ -0,0 +1,131 @@
+/* global page, maxCLS, stepNumber, testMultiple, maxDCL, maxFCP, maxLCP, passRatio */
```

Comment:
I saw no significant difference in performance results.
I wondered how about when there is more Items.
I wish there was another test with 100 Items.